docs/console.rst

```diff
@@ -1,62 +1,74 @@
+.. meta::
+   :last-reviewed: 2020-08-23
+
 .. _sql-console:
 
 ===========
 SQL console
 ===========
 
-The CrateDB admin UI comes with an SQL console so that you can execute queries
-against your cluster directly from your web browser.
+The :ref:`CrateDB Admin UI <index>` comes with an *SQL console* so that you can
```

Comment:
Have we decided on whether it should be "a SQL" or "an SQL" ?

Reply:
I am not sure we came to a conclusion. however, based on https://english.stackexchange.com/a/5266, I currently believe that "an SQL" is the best choice